server/src/main/java/org/opensearch/search/backpressure/stats/SearchTaskStats.java

```diff
@@ -33,13 +34,13 @@ public class SearchTaskStats implements ToXContentObject, Writeable {
     private final long cancellationCount;
     private final long limitReachedCount;
     private final long completionCount;
-    private final Map<TaskResourceUsageTrackerType, TaskResourceUsageTracker.Stats> resourceUsageTrackerStats;
+    private final Map<TaskResourceUsageTrackerType, Stats> resourceUsageTrackerStats;
```

Comment:
I will suggest to use:

* TaskResourceUsageTracker.Stats as Stats itself is not very clear
* `TaskResourceUsageTracker` instead of `TaskResourceUsageTrackers.TaskResourceUsageTracker` for better readability

Essentially, add the imports in a manner such that code does not require inline changes

Reply:
That is a good point from contextual perspective for these abstracts. Thanks! will make the change